hyper/contrib.py

```diff
@@ -27,6 +27,7 @@ class HTTP20Adapter(HTTPAdapter):
     def __init__(self, *args, **kwargs):
         #: A mapping between HTTP netlocs and ``HTTP20Connection`` objects.
         self.connections = {}
+        self.force_proto = kwargs.get('force_proto')
```

Comment:
You got it. Would you rather I edit and squash the commit, or just add a new one?